src/planners/bidirectional.jl

```diff
@@ -0,0 +1,130 @@
+export BiDirectionalPlanner, BiGreedyPlanner, BiAStarPlanner
+
+"BiDirectionalPlanner best-first search planner."
+mutable struct BiDirectionalPlanner{F,B} <: Planner
```

Comment:
I think I would prefer this to be renamed `BidirectionalPlanner`, just because that's a bit more standard English-wise, though I recognize this means it would be less consistent with `BiGreedyPlanner` and `BiAStarPlanner`.

Reply:
Done